src/main/java/io/fusionauth/load/JavaHTTPLoadTestWorker.java

```diff
@@ -26,25 +34,62 @@
  * @author Daniel DeGroff
  */
 public class JavaHTTPLoadTestWorker extends BaseWorker {
+  private final HttpClient javaHTTPClient = HttpClient.newBuilder()
+                                                      .connectTimeout(Duration.ofSeconds(10))
+                                                      .followRedirects(Redirect.ALWAYS)
+                                                      .build();
+
+  private final String restClient;
+
   private final String url;
 
   public JavaHTTPLoadTestWorker(Configuration configuration) {
     super(configuration);
     this.url = configuration.getString("url");
+    this.restClient = configuration.getString("restClient");
   }
 
   @Override
   public boolean execute() {
-    ClientResponse<String, String> response = new RESTClient<>(String.class, String.class)
-        .url(this.url)
-        .bodyHandler(new ByteArrayBodyHandler("This is a body that is hashed".getBytes()))
-        .connectTimeout(7_000)
-        .readTimeout(7_000)
-        .successResponseHandler(new TextResponseHandler())
-        .errorResponseHandler(new TextResponseHandler())
-        .get()
-        .go();
-    return response.wasSuccessful();
+    if ("restify".equals(restClient)) {
+      ClientResponse<String, String> response = new RESTClient<>(String.class, String.class)
+          .url(this.url)
+          .setHeader("Content-Type", "text/plain")
+          .bodyHandler(new ByteArrayBodyHandler("This is a body that is hashed".getBytes()))
+          .connectTimeout(15_000)
+          .readTimeout(15_000)
+          .successResponseHandler(new TextResponseHandler())
+          .errorResponseHandler(new TextResponseHandler())
+          .get()
+          .go();
+
+      if (response.wasSuccessful()) {
+        return true;
+      }
+
+      if (response.exception != null) {
+        System.out.println(response.exception.getClass().getSimpleName() + " [" + response.exception.getMessage() + "]");
```

Comment:
Any reason we're using stdout directly instead of a logging framework?

Reply:
No particular reason. This is essentially a CLI tool, and most of the output is all using `stdout` for reporting, etc.  

We could consider moving all of the `stdout` to a logger config. 

In practice, this is really just for debugging - looking for `SocketExceptions` based upon some server configuration I was testing. I could probably just delete these lines.